Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -1481,6 +1485,87 @@ void Renderer::Swap(u32 xfb_addr, u32 fb_width, u32 fb_stride, u32 fb_height, u6
   }
 }
 
+void Renderer::UISwap()
```

Comment:
`UISwap` contains a bunch of copy/pasted code from `Swap`
This common code should be extracted out into a common function that both will call.